fifteen_min/index.html

```diff
@@ -5,9 +5,18 @@
     <script type="module">
         import { default as init } from './fifteen_min.js';
 
-        function isWebGL2Supported() { 
+        function isWebGL1Supported() {
```

Comment:
There are identical changes for our various html loaders. 

I'm almost frustrated enough to build an actual template/render solution to avoid all these dupe edits... almost..

Reply:
I wonder what point that'll make sense. Maybe just a simple script that any widgetry client might want to use, taking the title and `fifteen_min_bg.wasm` bit and creating the file. (And maybe even anything loading a full-screen wasm app that requires webgl)